react-delegationdashboard/src/components/PendingUndelegatedListView/index.tsx

```diff
@@ -0,0 +1,85 @@
+
+import { QueryResponse } from '@elrondnetwork/erdjs/out/smartcontracts/query';
+import * as React from 'react';
+import { useContext } from '../../context';
+import { UndelegatedValueType } from '../../helpers/types';
+import { contractViews } from '../../contracts/ContractViews';
+import denominate from '../Denominate/formatters';
+import UndelegatedValueRow from './UndelegatedValueRow';
+
+const UndelegatedListView = () => {
+    const { dapp, address, denomination, decimals, delegationContract } = useContext();
+
+    const [userUnstakeValue, setUserUnstakedValue] = React.useState(Array<UndelegatedValueType>());
+
+    const denomintateValue = (value: string): string => {
+        return denominate({ input: value, denomination: denomination, decimals: decimals, showLastNonZeroDecimal: false });
```

Comment:
For denomination you can skip the name, it will make the code shorter:
```
return denominate({ denomination, decimals, input: value, showLastNonZeroDecimal: false });
```